libraries/singleton-devices/Handler.cc

```diff
@@ -88,7 +88,7 @@ bool Handler::setDevice(std::string deviceDatabaseKey, yarp::dev::PolyDriver* de
         //device does not exists. Add to map
         if (!m_devicesMap.insert(std::pair<std::string, yarp::dev::PolyDriver*>(deviceDatabaseKey, device2add)).second) 
         {
-            yError() << " Error in GazeboYarpPlugins::Handler while inserting a new device pointer!";
+            yError() << " Error in gz-yarp-Plugins::Handler while inserting a new device pointer!";
```

Comment:
Can you review the error messages in this file? `gz-yarp-Plugins::Handler` sounds strange, probably we can just put namespace and name of the class?